packager/media/formats/mp4/box_definitions.cc

```diff
@@ -1709,6 +1709,29 @@ size_t ElementaryStreamDescriptor::ComputeSizeInternal() {
   return HeaderSize() + es_descriptor.ComputeSize();
 }
 
+MHAConfiguration::MHAConfiguration() = default;
+MHAConfiguration::~MHAConfiguration() = default;
+
+FourCC MHAConfiguration::BoxType() const {
+  return FOURCC_mhaC;
+}
+
+bool MHAConfiguration::ReadWriteInternal(BoxBuffer* buffer) {
+  RCHECK(ReadWriteHeaderInternal(buffer) &&
+         buffer->ReadWriteVector(
+             &data, buffer->Reading() ? buffer->BytesLeft() : data.size()));
+  RCHECK(data.size() > 1);
+  mpeg_h_3da_profile_level_indication = data[1];
```

Comment:
Should this be inside an `if (buffer->Reading())` condition? i.e.

```
  if (buffer->Reading()) {
    RCHECK(data.size() > 1);
    mpeg_h_3da_profile_level_indication = data[1];
  }
```

Reply:
Yes, that is better I think, though it does not really matter.

Reply:
Ok, I just left it alone.